### ISSUE TITLE
Notebook Cells: Add ability to split/merge cells 

### ISSUE THREAD

**Issue by rajeshka** [microsoft/azuredatastudio#17169]
**Is your feature request related to a problem? Please describe.**
There is currently no easy way to edit/split new cells in ADS. in VSCode the cells can be split with a single click. In ADS it would require following steps
- Got to edit mode of the cell
- Select the text that needs to be split and Cut
- Create new cell 
- Paste the content
- Rinse and repeat if the content need to be split further

**Describe the solution or feature you'd like**
![image](https://user-images.githubusercontent.com/29285871/134949047-a771a062-4500-4231-8fce-0d7fbc3b14e8.png)

**Additional context**
Users creating TSGs are facing major friction and are questioning why the feature is not there when VSCode supports it.

**Comment by radentonev** [comment]
I think it's a good idea to have a "split cell" functionality. In general, I like the solution that VSCode has with the split cell button. 

<img width="262" alt="Screen Shot 2021-09-27 at 9 53 45 AM" src="https://user-images.githubusercontent.com/71295692/134952433-ace45e1c-3d1e-4510-95d5-4fb235dad83a.png">

Your screenshot is a bit ambiguous to me, since it shows a bunch of actions in a nested menu, but I assume your intention for this ticket is only to add the split cell button in the cell toolbar. I like that, however, I think it should not be the first action in the cell toolbar. I think the button to exit edit mode should remain first and the split cell can be placed after that.

Thanks,
Raden

**Comment by rajeshka** [comment]
> Your screenshot is a bit ambiguous to me, since it shows a bunch of actions in a nested menu, but I assume your intention for this ticket is only to add the split cell button in the cell toolbar. I like that, however, I think it should not be the first action in the cell toolbar. I think the button to exit edit mode should remain first and the split cell can be placed after that.

That is correct. The screenshot is from VSCode and shows 2 ways to invoke split cells. We can choose whichever is the right option for ADS without overcrowding the contextual menu

**Comment by Shirley-f-tan** [comment]
When TSGs are moved from OneNote to ADS, it's usually one big cell per TSG, adding splitting a big cell into multiple cells, one cell per paragraph, will be very useful to convert some of text cells into code cells. Of course, merge will be also useful as well. Thanks!

**Comment by chlafreniere** [comment]
@rajeshka is this different than #8541, or can we close this as a dupe?

**Comment by rajeshka** [comment]
> @rajeshka is this different than #8541, or can we close this as a dupe?

Its a dupe of this.

**Comment by rajeshka** [comment]
https://github.com/microsoft/azuredatastudio/pull/17185 - PR for the above
